source/utils/themes.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+
+// Each component offered in the React-Polymorph library could have a Theme API
```

Comment:
change "could have a theme api" to "has a …" 😉

Reply:
Will do!